### ISSUE TITLE
[Idea] Send SDC directly from ShadowChat to recipient

### ISSUE THREAD

**Issue by allienworks** [shadowproject/shadow#48]
Just a suggestion, feel free to close:

Ability to send SDC to chat recipient (GetGems/IRC style) directly from chat, e.g.:

* 10 SDC (will send 10 SDC without coin control)
* !rain 10 (to all participants in group discussion)

Etc.

**Comment by dasource** [comment]
@allienworks 
This brings up a discussion we are having internally.

I am of the opinion that Chat address should not be ideally used for receiving/sending because
1. If you are not paying attention you could chat with an address you planned to keep private thus exposing your balance 
2. This could be used easily to spam and/or extort from users (you know the balance, lets spam the user type scenarios).
3. If you use a stealth address and/or multiple receive addresses i.e. lets say 100 ... you have a dropdown of 100 addresses to choose from to chat with - not very user friendly.

My proposal is to split out the Chat address AKA Umbra Identity to a separate BIP32 path outside of the current Account BIP44 path.
This will allow you to create Identities which can be used for both Chat and Market.
This will also still allow you to receive SDC to these address (should you so desire) but you will not by default have the ability to send from this address (directly from GUI).

**Comment by allienworks** [comment]
> My proposal is to split out the Chat address AKA Umbra Identity to a separate BIP32 path outside of the current Account BIP44 path.

This separation of identity and funds sounds like a resonable solution _if_ this will be implemented (assuming I understood it well). And if those "ID/chat addys" will forward any incoming cash sent there (to your "cash addys"), that would be great.

**Comment by kewde** [comment]
In any case we must use stealth addresses here. 
Next weeks update has a JSON format update planned that should open the gap to use stealth addresses in the future, altough it will initially not support this, it may be used to support it later.

**Comment by dasource** [comment]
> This separation of identity and funds sounds like a resonable solution if this will be implemented (assuming I understood it well). 
Yes it is in the works .... Lots of privacy benefits and is more secure to have it that way.

>And if those "ID/chat addys" will forward any incoming cash sent there (to your "cash addys"), that would be great.
Thinking about it who says you might not want to send some SDC on your Identity account for the pure joy of using `!rain` - keeping it all nice and separated.
